src/dialogs/more-info-dialog.html

```diff
@@ -19,6 +19,7 @@
       }
 
       state-history-charts {
+        width: 100%;
```

Comment:
This will not fix it.

State history charts is already `display: block` so has a width of 100%.